src/store/modules/ADempiere/browserManager.js

```diff
@@ -61,7 +61,7 @@ const browserControl = {
       isLoadedContext = false,
       selectionsList = [],
       pageNumber = 1,
-      pageSize = ROWS_OF_RECORDS_BY_PAGE
+      pageSize = 50
```

Comment:
change `ROWS_OF_RECORDS_BY_PAGE` constants to 50.

Reply:
y If a new constant is created since it is used in many request which will affect the behavior